packages/grafana-ui/src/components/FileDropzone/FileDropzone.tsx

```diff
@@ -165,42 +161,12 @@ export function FileDropzone({ options, children, readAs, onLoad, fileListRender
     return <FileListItem key={file.id} file={file} removeFile={removeFile} />;
   });
 
-  const setErrors = (rejectedFiles: FileRejection[]) => {
```

Comment:
Wondering why the error handling/display has been moved out of this component? I think it's pretty useful when the component itself takes care of error handling/displaying them.